govcd/catalog_test.go

```diff
@@ -788,3 +788,38 @@ func cleanupCatalogOrgVdc(check *C, sharedCatalog Catalog, vdc *Vdc, vcd *TestVC
 	err = adminOrg.Delete(true, true)
 	check.Assert(err, IsNil)
 }
+
+// Tests System function UploadOvf by creating catalog and
```

Comment:
I think this description is incorrect, because you're testing UploadOvfByLink here.

Reply:
updated